resources/syntax/Twig.sublime-syntax

```diff
@@ -66,7 +66,7 @@ contexts:
       push: statement_block_meta
 
   statement_block_meta:
-    - meta_scope: meta.statement.twig
```

Comment:
Why adding `meta.template` here?

`meta.template` should be used to scope a whole document being a template, only.

Evaluated sections "template tags" such as `{% ... %}` should be scoped `meta.embedded` to indicate embedded code being evaluated by the template engine in an otherwise passive text document.

Text interpolations like `#{...}` would receive `meta.interpolation`.

Reply:
Solution is to remove clear_scope from normal blocks and provide a separate context to be included into strings, which clears just 1 scope.

Reply:
This is not related to interpolation and strings. This is an embedded language.